doc/sphinx/examples/input/index.md

```diff
@@ -1,13 +1,26 @@
 # Example Input Files
 
-The following example input files, demonstrate how some of the features of Cantera's
-YAML format can be used to define phases, species, and reactions. These files are a
-subset of the input files included with Cantera.
+:::{admonition} Disclaimer
+Input files distributed with Cantera are provided for **illustration purposes only**.
+Users should consult the primary literature to identify suitable mechanisms before
+making a selection for research or applications. The Cantera maintainers assume no
+responsibility for uncritical usage of these example files or conclusions drawn from
+them.
```

Comment:
I don't think we need the statement about responsibility. The license already disclaims suitability for a particular purpose, etc. Insofar as this is a CYA statement, one could uncharitably read "uncritical usage" and interpret it to mean that we are responsible for critical usage, when really we aren't responsible for _any_ usage 😊 Besides, it strikes IMO a very defensive tone that doesn't feel great to me